addon/ng2/blueprints/ng2/files/__path__/test.ts

```diff
@@ -0,0 +1,41 @@
+
+/*global jasmine, __karma__, window*/
+require('core-js/es6');
```

Comment:
So because of instrumentation and coverage, by switching to imports the coverage reporter will see these as errors. So I can change these to imports, but then I'll turn off coverage generation and leave it as a plus task when instrumentation and coverage catch up with typescript and es6 and sourcemapping. 

Thoughts?